### DISCUSSION TITLE
Not able to connect putty or AWS Cloudshell

### DISCUSSION THREAD

**Discussion by LavanyaRamesh319** [learndevopsonline/learndevopsonline#1853] [Q&A]
I am from batch 72. I am unable to connect to putty or AWS cloudshell. It is showing as "Access denied". I have given login as "root" and password as "Devops321"
Could you please tell me how to resolve it?

**Comment by chaithanya1812** [comment]
change login as centos, it will work and also password is ;- DevOps321
